trayer: add page (#10441)
tldr-pages/tldr: pages/linux/trayer.md

# trayer

> > A lightweight GTK-2 based systray for UNIX desktop.

Reviewer: ```suggestion
> A lightweight GTK-2 based systray.
```
IMO we don't need to specify `for UNIX desktop` as we are moving away from this naming in other platform pages.

> More information: <https://github.com/sargon/trayer-srg>.

> - Run trayer:

Reviewer: ```suggestion
- Run `trayer`:
```

`trayer`

> - Position trayer to the top:
> 
> `trayer --edge {{top}}`

Reviewer: ```suggestion
- Position `trayer` to a specific edge:

`trayer --edge {{left|right|top|bottom}}`
```

> - Provide height as 32px and width as 10% of screen resolution:

Reviewer: ```suggestion
- Provide a specific height and width of the panel (in pixels):
```
I would suggest having the command description a bit more generalised

> `trayer --width {{10}} height {{ 32 }}`

Reviewer: ```suggestion
`trayer --width {{10}} --height {{32}}`
```
Shouldn't the height be a flag too here?

Author: yes... I forgot. Thanks for that

> - Provide width in pixel:

Reviewer: ```suggestion
- Provide the width of the panel in pixels or percentages:
```

> `trayer --widthtype {{pixel}} --width {{72}}`

Reviewer: ```suggestion
`trayer --widthtype {{pixel|percent}} --width {{72}}`
```

> - Align trayer to the right:
> 
> `trayer --align {{right}}`

Reviewer: ```suggestion
- Align `trayer` to a specific direction:

`trayer --align {{left|center|right}}`
```

> - Provide icon gap 10px:

Reviewer: ```suggestion
- Provide space between icons (in pixels):
```

> - Provide space between icons (in pixels):

Reviewer 2: ```suggestion
- Provide spacing between icons (in pixels):
```
